build.gradle

```diff
@@ -470,6 +470,11 @@ project(':iceberg-mr') {
       exclude group: 'org.apache.calcite.avatica'
     }
   }
+
+  test {
+    // testJoinTables / testScanTable
+    maxHeapSize '1500m'
```

Comment:
This is the fix for flaky tests? What was the heap size before?

Reply:
The heap size was not set before. So I assume it was the default.
Based on this [doc](https://docs.gradle.org/5.0/userguide/upgrading_version_4.html#rel5.0:default_memory_settings), the default is 512MB

I have checked the other tests, and found we use 1500/2500 in some cases. I figured that 1.5G should be enough as a first try, and it looks like it solved the flakiness 😄